### PR TITLE
THAT FORGOTTEN BIG CHAR SETUP UPDATE (You guys have added 2 more conflict PRs while leaving this one untouched smh.)

### PR DESCRIPTION
**Pull request by Verkister**
**MAJOR HIGHLIGHTS:**
**-> A WHOLE BUNCH OF NEW MARKINGS AND CHARACTER ASSETS.
-> DUAL COLORABLE TAILS AND TAURS.
-> INDIVIDUALLY COLORABLE EARS.
-> FIXED CHARACTER SETUP PREVIEWS.
-> A WHOLE BUNCH OF MINOR VISUAL TWEAKS.**

**tl;dr chronological changelog details below.**

-Added a bunch of marking assets from my older citadel ones.(otie, cow, corvid)
-Ported over the colorable otie wagtail.
-Added ears for otie and cow, colorable and plain versions.
-Ported over my old citadel drake taur. (disabled for now)
-Added a fixed _vr version of ponytail 6.
-Fixed the pixel offset issue with antlers.
-Added one pixel to every wolf taursuit to match them with the earlier body tweak that smoothed out the 90 degree tail snap angle.
-Added a few bootleg raccoon marking bits.
-Ported otie taur by request sure why not. (disabled for now)
-Tweaked a new properly centered curltail middle icon.
-Lined up and un-overlapped the setup previews. Also taurs can now fit into the frame and not be cropped in half.
-Unlocked the hidden and forgotten feature for secondary tail/taur colors.
-<s>Currently the only enabled things are the colorable wolf tail and a new dual color wolftaur.</s> The framework for more is ready for use though!
-Liberated ear coloring from the oppression of hair color.
-Disabled drake and otie taurs since they lack all gear compatibility :v
-Made dual-color wagtails possible.
-Added extra color features to curltail(wag), vulptail(wag), stripeytail, tigertail(wag), ringtail, naga, lizardtaur, felinetaur.
-As with the wolftaur earlier, naga, lizard, and felinetaurs have dual-color versions as subtypes to prevent compatibility issues with custom taurs.
-Merged ShadowLarkens' Big snake tail into the sprite file to avoid destructive PR conflicts.
-Added ArgobargSoup's zorren sprite stuffs ready to be coded in.
-Fixed a typo that probably caused the vulp shorttail wagstate be broken.
-Redirected zorren species towards the new icon file.
-Added colourable elf ears.
-Added framework for custom heads using custom eyes without messing the base code.
-Slightly tweaked sergal markings.
-Added new eyes for sergals with little more snout space inbetween.
-Added teshari feathers as markings after failing to fix them for custom species. |

### CODE REVIEW

**Comment by Yoshax** [comment]
A lot of that's kinda... Vague. Like, what is 'fixed'?

**Comment by Verkister** [comment]
Pesky stray pixels and accessory sprites being out of their right place. Just taking a visual pebble out of my shoe there heh

**Comment by Verkister** [comment]
I'd say this PR is done for now unless more changes or additions are requested.

**Comment by Verkister** [comment]
Someone's gonna have to merge the snake tail of the other PR manually into the sprite file. The more ya keep these PRs lingering, the more there will be file conflicts.

**Comment by SpadesNeil** [comment]
You need to stop adding stuff to a single PR. It just delays the review process. In future, when you make a PR, focus on one task. Do another PR for different things. Otherwise it becomes "out of scope" which is bad and confusing.

Example: I'm doing a PR to fix the map. Suddenly I find a problem with guns. Unless fixing it pertains to my current task, I'll fix it in another PR.

**Comment by Verkister** [comment]
Leaving the PRs lingering untouched for several days delays the process a lot more imo. I wouldn't have needed to add anything past the 4 days old commits if there wasn't a destructive file conflict threat imminent.

**Comment by Arokha** [comment]
🤔

**Comment by Verkister** [comment]
Yeah all's been fixed now and y'all are now free to do yer review magics and decide whether we get the good shit in sooner or later.

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm [hunk 1]
```diff
@@ -133,7 +133,7 @@
 /datum/species/hi_zoxxen
 	name = "Highlander Zorren"
 	name_plural = "Zorren"
-	icobase = 'icons/mob/human_races/r_fox.dmi'
+	icobase = 'icons/mob/human_races/r_fox_vr.dmi'
```
Comment (SpadesNeil) [inline code/modules/mob/living/carbon/human/species/station/station_vr.dm]: Um, did you rename these .dmi files? Because those species are ours.

**Comment by kcin2001** [comment]
i want it then i can make my sergaltaur because that the very least i can have the 2 tone taur half

**Comment by SpadesNeil** [comment]
Has this been tested?

**Comment by Verkister** [comment]
Tested working at my end, not sure if anyone else's attempted yet.

**Comment by Verkister** [comment]
Couple eyesore shots from the test runs during development (i think the taur ears pic there has cyan ears instead of white but that slipped past the radar due to my colorblindness and was caused by the already fixed r_ear/red_ear var conflict lmao)
a4586e144ed1268c52b3233cffa19561 1e12911769ad0afac755d68d92b9d11c 1c07e61047b15da9dbe0c391d504b0cf

**Comment by SpadesNeil** [comment]
Can we just merge this complete pizdetz already? :U

**Comment by SpadesNeil** [comment]
nope, another conflict
rustle

**Comment by Verkister** [comment]
fennix tail stuff added to the conflict file so it should be fine for now

**Comment by chaoko99** [comment]
Did anything like this ever get added?

**Comment by Cyantime** [comment]
Yes, it was re-made and merged at #1905

**Pull request closed without merging**